### ISSUE TITLE
[How to] make a line follow by a give "name" of a rect.

### ISSUE THREAD

**Issue by MichelDiz** [konvajs/react-konva#280]
e.g 
getPos('my rect with unique name') #x,y

There is something similar?

Cheers.

**Comment by lavrton** [comment]
@MichelDiz can you clarify what do you find? Find a rectangle on the stage by the name?

**Comment by MichelDiz** [comment]
Yes, any obj on stage or layer.

I'm trying to create Arrows that follow objects dynamically (one to many). However, there will be groups that will or will not have Arrows. Using React-konva I could not get an easy way to do this. However, pure Konva there is how to do easily.

Basically the groups will be on one layer and the Arrows on a separate layer.

**Comment by lavrton** [comment]
You can take a reference to a Konva node, and then call `getStage()`:

```javascript
// in your lifecycle methods:
const stage = this.shape.getStage();
const rect = stage.findOne('.rectangle-selector');


// somewhere in render:
<Shape ref={node => { this.shape = node }} />
```

But usually it is not required in React app. If you are writing React app you should update state of the app with objects movement. State store information about position. Then Arrow component will update its props from app state.

**Comment by MichelDiz** [comment]
I do not know why (Maybe it's because I'm using TypeScript), I could not get your code to work. However, I tried using the click handler.
```js
onMouseMove = { e => {
const test = e.target.getStage().find('Label')
console.log(test)
}
```
log:
![image](https://user-images.githubusercontent.com/6570955/47892618-845c2900-de37-11e8-8be2-d2eaf339dd43.png)

I had given a rectangle before as a figure of speech. In fact I intend to "follow" (or "anchor") in different objects. As Label ====> Group.

What I have did above is that I can filter the labels that exist. The problem is that I select by index. I would like to select by name. In the Log I put "MICHEL" in its attribute "attrs: {x: 3, and: 100, name:" MICHEL "}

There's no e.g:

```js
const test = e.target.getStage().find('Label').getByName('MICHEL')
const pos = test.position();
```
